.github/workflows/end2end.yaml

```
@@ -0,0 +1,400 @@
+---
+name: build-iso-and-end2end-test
+
+concurrency:
+  group: "zenko-tests-${{ github.ref_name }}"
+  cancel-in-progress: true
+
+on:
+  push:
+    branches:
+      - "documentation/**"
+      - "feature/**"
+      - "improvement/**"
+      - "bugfix/**"
+      - "w/**"
+      - "hotfix/**"
+      - "ci-dev/**"
+      - "user/**"
```

Just noticed a queue branch missing, make sure it's available on other workflows you expect to be triggered before merging (if any)
```suggestion
on:
  push:
    branches:
      - "documentation/**"
      - "feature/**"
      - "improvement/**"
      - "bugfix/**"
      - "w/**"
      - "hotfix/**"
      - "ci-dev/**"
      - "user/**"
      - "q/*/**"
```

---

Oh thanks! I will add that.

---

Added in https://github.com/scality/Zenko/pull/1717/commits/fa83c2be495163f23e8d9c0d5437f606af9f61b4